src/parser/Statement.ts

```diff
@@ -619,12 +621,23 @@ export class IfStatement extends Statement {
     public readonly condition: Expression;
     public readonly thenBranch: Block;
     public readonly elseBranch?: IfStatement | Block;
-    public readonly isInline?: boolean;
 
     public readonly kind = AstNodeKind.IfStatement;
 
     public readonly range: Range | undefined;
 
+    get isInline() {
```

Comment:
This could potentially be a `utils` function that is more general - i.e. it could work for all AstNodes, by iterating all tokens and properties that are also AstNodes